pkg/controller/ec2/vpcendpoint/setup.go

```diff
@@ -0,0 +1,176 @@
+package vpcendpoint
```

Comment:
can you add unit tests for this file? look for files called `setup_test.go` for examples, like https://github.com/crossplane/provider-aws/blob/master/pkg/controller/docdb/dbsubnetgroup/setup_test.go

Reply:
Unit tests are added.